Description: Fix NWChem input extension input file generation for the
 CCSD method.
Author: Daniel Leidert <daniel.leidert@wgdd.de>
Forwarded: no
Last-Update: 2011-04-25

--- a/libavogadro/src/extensions/nwcheminputdialog.cpp
+++ b/libavogadro/src/extensions/nwcheminputdialog.cpp
@@ -439,6 +439,7 @@
         mol << "ccsd\n";
         mol << "  # Exclude core electrons from CCSD treatment\n";
         mol << "  freeze atomic\n";
+        mol << "end\n\n";
       default:
       case RHF:
           break;
